core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -158,6 +158,21 @@ case class DriverLogUnsupportedOperators(
   }
 }
 
+case class StatusProfileResult(
+    path: String,
+    status: String,
+    message: String = "") extends ProfileResult {
+    override val outputHeaders: Seq[String] = Seq("Event Log", "Status", "Description")
+
+    override def convertToSeq: Seq[String] = {
+      Seq(path, status, message)
+    }
+
+    override def convertToCSVSeq: Seq[String] = {
+      Seq(path, status, message)
+    }
+  }
+
```

Comment:
I like that you tried to do some refactor in this PR to make the code reusable.

If we define a new `StatusProfileResult`, then we have two classes

- `QualificationAppInfo.StatusSummaryInfo`: which has an extra fields like appID
- `ProfileClassWarehouse.StatusProfileResult`

One of the above should go away because they do the same thing.
So, we should add the extra field appID to `StatusProfileResult`, then replace the usage of `StatusSummaryInfo` with `StatusProfileResult`.

Reply:
Thanks! Added `appID` in `StatusProfileResult` and removed `StatusSummaryInfo`